### PR TITLE
LOOFAH-9

### PR DESCRIPTION
**Pull request by Caitlin-cooling**
This PR implements the homepage and a basic skills page that displays hard coded skills. 

Note that it removes a lot of crap from react create app.

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        "ecmaFeatures": {
+            "jsx": true
+        },
+        "ecmaVersion": 2018,
+        "sourceType": "module"
+    },
+    "plugins": [
+        "react"
+    ],
+    "rules": {
```
Comment (Caitlin-cooling) [inline .eslintrc.js]: If anyone has any other eslint rules that they want to add let me know, these are just common ones that we use on leap.

File: src/Skills/Skills.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import React, { useState } from 'react';
+import Skill from '../Skill/Skill';
+
+const Skills = () => {
+  const [skillsState] = useState({
+    skills: [
+      {title: 'JavaScript', description: 'Learn it'},
+      {title: 'Java', description: 'Keep learning it'}
+    ]
+  });
```
Comment (Caitlin-cooling) [inline src/Skills/Skills.js]: Will be using GraphQL to get the skills when LOOFAH-3 is done, felt that this was the most simple way to get something  together to start with.

File: src/App.test.js [hunk 3]
```diff
@@ -1,9 +1,26 @@
-import React from 'react';
-import { render } from '@testing-library/react';
 import App from './App';
+import HomePage from './HomePage/HomePage';
+import Skills from './Skills/Skills';
+import React from 'react';
+import { mount } from 'enzyme';
+import { MemoryRouter } from 'react-router';
+
+it('/ route goes to home page', () => {
```
Comment (Learner5200) [inline src/App.test.js]: Nitpick:

Reading up it looks like `it` is an alias of `test`, useful when we want to approximate english syntax when describing the behaviour of a particular entity/method, is that right? In this case I wonder if `test` would be more readable, as "it / route goes to home page" pulled me up short a bit.

File: src/Skill/__tests__/Skill.test.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { mount } from 'enzyme';
+import React from 'react';
+import Skill from '../Skill';
+
+it('renders the title and description of given given', () => {
```
Comment (Learner5200) [inline src/Skill/__tests__/Skill.test.js]: typo ("given given")

File: src/App.test.js (hunk 3 above)
Comment (Caitlin-cooling) [reply src/App.test.js]: Excellent point! I'm used to using `it` in my tests so didn't give it a second thought, but will change to `test`

**Pull request merged**